apps/meeting/CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing Guidelines
+
+Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional
+documentation, we greatly value feedback and contributions from our community.
+
+Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
+information to effectively respond to your bug report or contribution.
+
+
+## Reporting Bugs/Feature Requests
+
+We welcome you to use the GitHub issue tracker to report bugs or suggest features.
+
+When filing an issue, please check existing open, or recently closed, issues to make sure somebody else hasn't already
```

Comment:
[nit] would it be better to remove the commas?
`...please check existing open or recently closed issues to make...`

Reply:
Removed from this and the main contributing guide as well. I had just copied over from it 😅 .